healthcare/controllers/queries.py

```diff
@@ -4,14 +4,10 @@
 @frappe.whitelist()
 @frappe.validate_and_sanitize_search_inputs
 def get_healthcare_service_units(doctype, txt, searchfield, start, page_len, filters):
-	query = """
-		select name
-		from `tabHealthcare Service Unit`
-		where
-			is_group = 0
-			and company = {company}
-			and name like {txt}""".format(
-				company = frappe.db.escape(filters.get('company')), txt = frappe.db.escape('%{0}%'.format(txt)))
+	table = frappe.qb.DocType("Healthcare Service Unit")
+	query = frappe.qb.from_(table).where(table.is_group == 0) \
+	.where(table.company == frappe.db.escape(filters.get('company'))) \
+    .where(table.name.like(frappe.db.escape('%{0}%'.format(txt)))).get_sql()
```

Comment:
@gavindsouza  will `frappe.db.escape` cause a similar problem?
ref -> https://github.com/frappe/frappe/issues/15087

Reply:
All user inputs should be escaped, but they weren't here before. @ChillarAnand Any reason for this?

---

> will frappe.db.escape cause a similar problem?

If you can't anticipate it, DIY. Nothing better than good ol' trying it out yourself :trollface:

Reply:
So doesn't seem like a problem